ftw/catalogdoctor/tests/test_surgery_remove_from_unindex.py

```diff
@@ -0,0 +1,135 @@
+from datetime import date
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.catalogdoctor.compat import DateRecurringIndex
+from ftw.catalogdoctor.surgery import RemoveFromUnIndex
+from ftw.catalogdoctor.tests import FunctionalTestCase
+from ftw.catalogdoctor.utils import find_keys_pointing_to_rid
+from Products.PluginIndexes.DateIndex.DateIndex import DateIndex
+from Products.PluginIndexes.FieldIndex.FieldIndex import FieldIndex
+from Products.PluginIndexes.KeywordIndex.KeywordIndex import KeywordIndex
+
+
+class TestRemoveFromUnIndex(FunctionalTestCase):
+
+    def setUp(self):
+        super(TestRemoveFromUnIndex, self).setUp()
+
+        self.grant('Contributor')
+        self.folder = create(Builder('folder').titled(u'Foo'))
+
+        # pretend we are something that supports recurring dates
```

Comment:
I'd move that to the only test that needs it, namely `test_remove_healthy_object_from_daterecurringindex`